pkg/controllers/provisioning/provisioner.go

```diff
@@ -151,32 +143,27 @@ func (p *Provisioner) launch(ctx context.Context, constraints *v1alpha5.Constrai
 	if err := p.Spec.Limits.ExceededBy(latest.Status.Resources); err != nil {
 		return err
 	}
-	errs := make([]error, packing.NodeQuantity)
-	workqueue.ParallelizeUntil(ctx, packing.NodeQuantity, packing.NodeQuantity, func(i int) {
-		errs[i] = p.create(ctx, &cloudprovider.NodeRequest{Constraints: constraints, InstanceTypeOptions: packing.InstanceTypeOptions}, packing.Pods[i])
-	})
-	return multierr.Combine(errs...)
-}
 
-func (p *Provisioner) create(ctx context.Context, nodeRequest *cloudprovider.NodeRequest, pods []*v1.Pod) error {
-	node, err := p.cloudProvider.Create(ctx, nodeRequest)
+	nodeRequest := &cloudprovider.NodeRequest{Constraints: &node.Constraints, InstanceTypeOptions: node.InstanceTypeOptions}
```

Comment:
If I don't pass the node, I can't use the node's `String()` method.  If I pass them both, it's a bit redundant since the node request is just a subset of the node.